### PR TITLE
New market fixes

### PR DESCRIPTION
**Pull request by koredefashokun**
## Description
This PR fixes issues related to new markets. Most of the fixes are around the way we fetch data (parsing the `marketKey` correctly from the `asset` field)

## Related issue
- #595 

## Motivation and Context
This PR solves issues with new markets (still currently paused on mainnet)

## How Has This Been Tested?
N/A

### CODE REVIEW

**Comment by platschi** [comment]
WTI market text and description are off:

Screen Shot 2022-04-07 at 14 08 43

File: components/Currency/CurrencyIcon/CurrencyIcon.tsx [hunk 1]
```diff
@@ -24,8 +24,20 @@ export type CurrencyIconProps = {
...
 	'https://raw.githubusercontent.com/Synthetixio/synthetix-assets/master/snx/SNX.svg';
 
-export const getSynthIcon = (currencyKey: CurrencyKey) =>
-	`https://raw.githubusercontent.com/Synthetixio/synthetix-assets/master/synths/png/${currencyKey}.png`;
+export const getSynthIcon = (currencyKey: CurrencyKey) => {
+	let parsedCurrencyKey = currencyKey as string;
+
+	// Using a switch so that we can add more currencies if the need arises.
+	switch (currencyKey as string) {
+		case 'sWTI':
```
Comment (Tburm) [inline components/Currency/CurrencyIcon/CurrencyIcon.tsx:32]: SIGH

File: utils/futures.ts [hunk 2]
```diff
@@ -16,3 +18,21 @@ export const getMarketKey = (asset: string | null, networkId: NetworkId) => {
 export const getDisplayAsset = (asset: string | null, networkId: NetworkId) => {
 	return asset ? (asset[0] === 's' ? asset.slice(1) : asset) : null;
 };
+
+export const getSynthDescription = (synth: string, synthsMap: Dictionary<Synth>, t: TFunction) => {
+	const parsedSynthKey = synth ? (synth[0] !== 's' ? `s${synth}` : synth) : '';
+
+	switch (parsedSynthKey) {
+		case 'sWTI':
```
Comment (Tburm) [inline utils/futures.ts:26]: BIG SIGH

**Review by Tburm — APPROVED** [review]
A real bummer we have to do it this way but I understand it's not really in our control.

**Pull request merged**